e2etests-cli/main_test.go

```diff
@@ -1005,6 +1005,19 @@ layers: {
 				assert.Equal(t, "x -> y\n", string(gotBar))
 			},
 		},
+		{
+			name: "fmt-check",
+			run: func(t *testing.T, ctx context.Context, dir string, env *xos.Env) {
+				writeFile(t, dir, "foo.d2", `a ---> b`)
+				writeFile(t, dir, "bar.d2", `x ---> y`)
+				err := runTestMainPersist(t, ctx, dir, env, "fmt", "--check", "foo.d2", "bar.d2")
+				assert.ErrorString(t, err, "failed to wait xmain test: e2etests-cli/d2: failed to fmt: exiting with code 1: found 2 unformatted files. Run d2 fmt to fix.")
+				gotFoo := readFile(t, dir, "foo.d2")
+				gotBar := readFile(t, dir, "bar.d2")
+				assert.Equal(t, "a ---> b", string(gotFoo))
+				assert.Equal(t, "x ---> y", string(gotBar))
```

Comment:
can you extend this to have `d2 fmt --check` run on formatted files?

Reply:
Done in 7cb58fc. I added a formatted file to this first test case, and also added another one where all the files are already formatted correctly.